### PR TITLE
Add caution to USB device

### PR DESCRIPTION
**Pull request by Yu-Jack**
- Related Issue https://github.com/harvester/harvester/issues/6720

### CODE REVIEW

File: docs/advanced/addons/pcidevices.md [hunk 1]
```diff
@@ -104,6 +104,10 @@ A `USBDevice` resource in Harvester represents a USB device on the node. USB dev
 
 USB passthrough is slightly different from PCI passthrough. For example, you can fully control a USB controller with four USB ports by creating a `PCIDeviceClaim`. However, you can also create a `USBDeviceClaim` to control only one USB port. The other three USB ports remain available to the node.
 
+:::caution
+Please detach the USB device from VM and disable it before unplugging it.
```
Comment (jillian-maroket) [inline docs/advanced/addons/pcidevices.md:108]:
```suggestion
Before you remove the USB device, detach it from the virtual machine and then disable passthrough on the **USB Devices** screen.
```

Comment (jillian-maroket) [reply docs/advanced/addons/pcidevices.md:108]: I used *disable passthrough* because the UI control is `Disable Passthrough` (not Disable Device).

**Review by jillian-maroket — COMMENTED** [review]
I reworded the note. Let me know if I misunderstood anything.

File: docs/advanced/addons/pcidevices.md (hunk 1 above)
Comment (Yu-Jack) [reply docs/advanced/addons/pcidevices.md]: Sounds good, thanks!

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**